STEERING-COMMITTEE.md

```diff
@@ -46,6 +49,40 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
```

Comment:
Given that this is in `STEERING-COMMITTEE.md` we should move things to `TRADEMARK-COMMITTEE.md` either after this merges, or at least after things settle down.

Reply:
I worry that it looks like we're trying to hide it if we bury it in some way. There should be a very brief place someone can go to understand the top level governance of the project; maybe a new top level that says the main governance structures in Knative are KSC, KTC, and TOC, and links each?

Reply:
a separate doc to describe the KTC would be nice indeed and a top level doc that explains the overall governance seems also good. my 2cts

Reply:
@anicolao this is mostly about consistency with what we have today.  We have `TECH-OVERSIGHT-COMMITTEE.md` and this, so including KTC here seems misplaced.  I do think that a little blurb to dispatch folks to the right place would be useful though (perhaps at the top of README.md?).

Reply:
With some top level dispatch, I think it's fine. I just don't want it to be in any way misleading.